source/github-helpers/api.ts

```diff
@@ -164,7 +164,7 @@ export const v3paginated = async function * (
 	}
 };
 
-export const v4 = mem(async (
+export const v4uncached = async (
```

Comment:
If anyone is reviewing this PR post-merge… naming suggestions still welcome 😅

Reply:
Happen to think its good